src/app/pipes/math/bytes.ts

```diff
@@ -4,10 +4,10 @@ import {isNumberFinite, isUndefined, applyPrecision} from '../helpers/helpers';
 @Pipe({name: 'bytes'})
 export class BytesPipe implements PipeTransform {
   private dictionary: Array<{max: number, type: string}> = [
-    { max: 1e3, type: 'B' },
-    { max: 1e6, type: 'KB' },
-    { max: 1e9, type: 'MB' },
-    { max: 1e12, type: 'GB' }
+    { max: 1024, type: 'B' },
+    { max: 1048576, type: 'KB' },
+    { max: 1073741824, type: 'MB' },
+    { max: 1.0995116e12, type: 'GB' }
   ];
 
   transform(value: number, precision?: number): string | number {
```

Comment:
Also don't forget to change the README.md file regarding the bytes section.


Thanks!!